### DISCUSSION TITLE
Forest roads visualized like road with no access

### DISCUSSION THREAD

**Discussion by drazde** [osmandapp/OsmAnd#13125] [Ideas]
Hi, in Trentino (Italy) many roads are "forest roads", and usually are not allowed access with the car.
For hiking use is useful know that roads becouse you have to parking before it start.

I would like see this road like road not allowed access

Forest roads are tagged mainly with
```
highway = track
forestry = yes
```

Or more specific, two type of forest road:
```
designation = IT-TN:strada_forestale_tipo_a
designation = IT-TN:strada_forestale_tipo_b
```
type A and type B are both restricted acces

 
The presets for this road are described here:
https://wiki.openstreetmap.org/wiki/Trentino/Strade_forestali_provinciali

**Comment by scaidermern** [comment]
`highway=track` + `forestry=yes` looks like wrong tagging to me. `forestry=yes` doesn't forbid car access. To forbid car (and all other motorized) access you have to set `motor_vehicle=forestry`.
